mockData/writeAllReviews.js

```diff
@@ -0,0 +1,23 @@
+const generateReview = require('./reviewGenerator.js');
+const helpers = require('./reviewsHelperData.js');
+// const allRestaurants = require('./basicRestaurantData.js');
```

Comment:
Not sure if you'll need this later but I'd suggest deleting unused and commented out code

Reply:
Will need that line of code when it's time to feed all of these reviews into the database. It's just a reminder to myself.